packages/suite/src/components/suite/AppNavigation/index.tsx

```diff
@@ -31,8 +117,20 @@ const Wrapper = styled.div<{ settingsWrapper: boolean | undefined }>`
         `};
 `;
 
+const KeepWidth = styled.div<{ maxWidth?: string | undefined; border: boolean }>`
```

Comment:
nit: 
` | undefined` is redundant here as there is already question mark. Please prefer `maxWidth?: string` syntax over `maxWidth: string | undefined` (as I see on many other places)

No need to change it right now, I just want to point it out.

Reply:
ok , thank you for explaining. fixed in 331a45e053a53457e46506ccff18f29dd5f5b5fc